### PR TITLE
Created an entry for ABI

### PR DESCRIPTION
**Pull request by Jetoson**

### CODE REVIEW

**Comment by AnneCYH** [comment]
Hi @Jetoson , thanks for your PR!

I apologise for having jumped ahead, but could you please pull the latest changes from this repo to your branch?

The contribution guidelines have been updated, and we require that you
1. Sign the Canonical contributor license agreement, and
2. Use signed commits only.

Please see the information + instructions here.
https://canonical-kernel-docs.readthedocs-hosted.com/en/latest/how-to/contribute/

This is standard practice across Canonical, but as mentioned, I jumped ahead, and I apologise for the inconvenience.

I hope that does not deter you from contributing :)

Please let me know if any help is needed.

I will put the review on hold for now until the CLA is signed + commits are signed.

We appreciate your effort, thanks!

**Comment by Jetoson** [comment]
@AnneCYH  sure, will do. Thank you for your quick response. What should I fill in the Canonical Project Manager or contact field?

**Comment by AnneCYH** [comment]
Hi @Jetoson ,

I see you've signed the CLA and setup signed commits; thanks!

I'll pull in a Kernel engineer to give this a look. 

Thank you!

File: docs/reference/glossary.md [hunk 1]
```diff
@@ -12,7 +12,10 @@ DKMS
   See the dkms manpages for more information.
 
 ABI
-  Application binary interface.
+  Application binary interface, or ABI defines a stable interface between user-space
```
Comment (tswhison) [inline docs/reference/glossary.md:15]: LGTM.  Just one nit..
```suggestion
  Application Binary Interface, or ABI defines a stable interface between user-space
```

Comment (Jetoson) [reply docs/reference/glossary.md]: Thank you very much for the feedback. As  a beginner, I'm  learning a lot about kernel documentation and open source contribution in general.

I have added the requested changes. Let me know for any improvements I should work on.

Comment (AnneCYH) [reply docs/reference/glossary.md]: Thanks @Jetoson , we're so glad to have you join us at the start of your open source journey.
Please continue to contribute if something interests you in our Kernel docs.

On a related note, Canonical has the Open Documentation Academy, which is a platform for you to connect with the Documentation team at Canonical, and also a place to start getting involved in open source through documentation.
Read more about the ODA here:
https://canonical.com/documentation/open-documentation-academy

There are also specially curated issues which anyone who is interested can contribute to here, as part of the ODA efforts:
https://github.com/canonical/open-documentation-academy/issues

Let me know if you have any questions.

Thanks!

- Anne -

File: docs/reference/glossary.md [hunk 2]
```diff
@@ -12,7 +12,10 @@ DKMS
   See the dkms manpages for more information.
 
 ABI
-  Application binary interface.
+  Application Binary Interface, or ABI defines a stable interface between userspace
```
Comment (AnneCYH) [inline docs/reference/glossary.md:15]: Apologies @Jetoson , I think you may have misread this.
The suggestion was to spell it out without a hyphen, "user space", similar to how the Linux kernel docs does it.

Do submit the change and squash the changes; i'll merge it right after.

Thanks!

**Pull request merged**